sdc/tests/test_series.py

```diff
@@ -53,6 +53,7 @@
 from sdc.tests.gen_test_data import ParquetGenerator
 
 from sdc.tests.test_utils import test_global_input_data_unicode_kind4 as test_global_input_data_unicode
```

Comment:
Ok, let's use `test_global_input_data_unicode_kind4` from this file or from `sdc.tests.test_utils` if you are ready to remove `test_global_input_data_unicode_kind4` from this file.